packages/qocamlbrowser/qocamlbrowser.0.2.10/opam

```diff
@@ -4,9 +4,10 @@ homepage: "https://github.com/Kakadu/QOcamlBrowser_quick/"
 bug-reports: "https://github.com/Kakadu/QOcamlBrowser_quick/issues"
 authors: "Kakadu"
 build: [
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
-  ["sh" "-exc" "PATH=/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH make"]
+  ["sh" "-exc" "PATH=/usr/local/opt/qt/bin:/usr/lib64/qt5/bin:/usr/lib/qt5/bin:$PATH ./configure"]
```

Comment:
There is a way to make these depend on the OS, so you could use the right PATH in darwin and linux separately. I'll try to dig an example

Reply:
See https://github.com/ocaml/opam-repository/blob/master/packages/conf-blas/conf-blas.1/opam#L9 for example